src/domain/reservation/manage/list/ManageReservationsList.js

```diff
@@ -68,6 +69,8 @@ const ManageReservationsList = ({
                     onEditReservation={onEditReservation}
                     onInfoClick={() => onInfoClick(reservation)}
                     reservation={normalizedReservation}
+                    userCanCancel={reservationUtils.canUserCancelReservation(reservation)}
```

Comment:
```suggestion
                    userCanDelete={reservationUtils.canUserCancelReservation(reservation)}
```

Reply:
i know that's dumb but `Cancel` should go along with `Cancel`..
Delete is not one of reservation state..

Reply:
I saw the `propTypes` in [ManageReservationsDropdown.js](https://github.com/City-of-Helsinki/varaamo/pull/1004/files#diff-dbe147be8b0229268f557adf4ec40370), they don't have `userCanCancel`, instead, they have `userCanDelete`.

Reply:
Hah, will fix 👍